pyvista/core/_validation/validate.py

```diff
@@ -544,7 +544,7 @@ def validate_transform3x3(transform, /, *, name="Transform"):
 
     Parameters
     ----------
-    transform : array_like | vtkMatrix3x3 | scipy.spatial.transform.Rotation
+    transform : RotationLike
```

Comment:
`RotationLike` here implies that only 3x3 rotation matrices are supported. But any 3x3 matrix is supported... not sure how to communicate this, maybe just a note in the docs?

Reply:
Yes, I think that will get the message to users.